src/modules/form/components/viewable/DynamicViewField.tsx

```diff
@@ -149,7 +152,12 @@ const DynamicViewField: React.FC<DynamicViewFieldProps> = ({
       }
       return <TextContent {...commonProps} />;
     case ItemType.Currency:
-      return <TextContent {...commonProps} renderValue={(val) => `$${val}`} />;
+      return (
+        <TextContent
+          {...commonProps}
+          renderValue={(val) => `$${formatNumberWithTrailingZeroes(val)}`}
```

Comment:
Looks like we already have a currency formatter implemented using `Intl.NumberFormat`

Can we just use that instead?
`import { formatCurrency} from '@/modules/hmis/hmisUtil';`
```suggestion
          renderValue={(val) => `${formatCurrency(val)}`}
```

Reply:
Whoops yes! thanks, forgot about that one